cocos/core/gfx/webgl/webgl-swapchain.ts

```diff
@@ -175,6 +175,14 @@ export function getExtensions (gl: WebGLRenderingContext) {
         if (WECHAT) {
             res.noCompressedTexSubImage2D = true;
         }
+
+        // HACK: for HUAWEI devices
```

Comment:
I think the comment is not accurate. It is for taobo platform on HWAWEI devices.

Reply:
updated